genesis/src/genesis/framework.rs

```diff
@@ -13,21 +12,23 @@ use {
     },
     move_vm_runtime::{
         module_traversal::{TraversalContext, TraversalStorage},
+        move_vm::MoveVM,
         native_extensions::NativeContextExtensions,
         session::Session,
     },
     move_vm_types::gas::UnmeteredGasMeter,
+    moved_state::State,
     once_cell::sync::Lazy,
     std::{collections::BTreeMap, fs, path::PathBuf},
     sui_framework::SystemPackage,
     sui_types::base_types::ObjectID,
 };
 
-pub const CRATE_ROOT: &str = env!("CARGO_MANIFEST_DIR");
+pub const CRATE_ROOT: &str = concat!(env!("CARGO_MANIFEST_DIR"), "/../moved");
```

Comment:
I saw this to get the workspace directory. It might be useful, but not necessary to use it here.
https://github.com/rust-lang/cargo/issues/3946#issuecomment-973132993

Reply:
Nice, well I prefer to keep it this way in this case because it works out of the box with no workarounds.